pages/coq-team.html

```diff
@@ -0,0 +1,486 @@
+    <div class="role">RM: 8.10</div>
+    <div class="components">Nix, micromega</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/assia_mahboubi.jpg" alt="Assia Mahboubi" width="200px"/></div>
+    <div class="name"><a href="http://people.rennes.inria.fr/Assia.Mahboubi/">Assia Mahboubi</a></div><div class="nickname">amahboubi</div>
+    <div class="position">Inria Researcher in the Gallinette Team, Nantes, France</div>
+    <div class="role"><a href="https://cordis.europa.eu/project/id/101001995">ERC FRESCO PI</a></div>
+    <div class="components">Ring, Mathematical Components</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/erikmd.png" alt="Érik Martin-Dorel" width="170px"/>
+    </div>
+    <div class="name"><a href="https://www.irit.fr/~Erik.Martin-Dorel/">Érik Martin-Dorel</a></div><div class="nickname">erikmd</div>
+    <div class="position">Assistant Professor at IRIT, Toulouse University, France</div>
+    <div class="components">Primitive Types, Docker</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/cpitclaudel.jpg" alt="Clément Pit-Claudel" width="170px"/>
+    </div>
+    <div class="name"><a href="http://pit-claudel.fr/clement/">Clément Pit-Claudel</a></div><div class="nickname">cpitclaudel</div>
+    <div class="position">PhD Student at MIT, MA, USA</div>
+    <div class="components">Documentation, Alectryon, Proof-General, Company-Coq</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/proux.jpg" alt="Pierre Roux" width="200px"/>
+    </div>
+    <div class="name"><a href="https://www.onera.fr/fr/staff/pierre-roux">Pierre Roux</a></div><div class="nickname">proux</div>
+    <div class="position">Researcher at Onera, Toulouse, France</div>
+    <div class="components">Primitive Types</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/pi8027.jpg" alt="Kazuhiko Sakaguchi" width="200px"/></div>
+    <div class="name">Kazuhiko Sakaguchi</div><div class="nickname">pi8027</div>
+    <div class="position">PhD Student at the University of Tsukuba, Japan</div>
+    <div class="components">Coercions, Extraction, Hierarchy Builder</div>
+  </div>   
+  <div class="person">
+    <div class="picture"><img src="/files/team/VincentSe.jpg" alt="Vincent Semeria" width="200px"/></div>
+    <div class="name">Vincent Semeria</div><div class="nickname">VincentSe</div>
+    <div class="position">Expert Software Developer at Finastra, Paris, France</div>
+    <div class="components">Reals</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/aspiwack.png" alt="Arnaud Spiwack" width="180px"/></div>
+    <div class="name">Arnaud Spiwack</div><div class="nickname">aspiwack</div>
+    <div class="position">Senior Architect at Tweag I/O, Paris, France</div>
+    <div class="components">Proof Engine</div>
```

Comment:
I suppose I also did the primitive integer compilation in the bytecode compiler. I'm not sure how it should appear. Maybe 
```suggestion
    <div class="components">Proof Engine, , <verb>vm_compute</verb></div>
```